app/queries/get_linked.rb

```diff
@@ -0,0 +1,20 @@
+module Queries
+  module GetLinked
+    def self.call(content_id, link_type)
+      linked_items = LinkedItem.where(target: content_id, link_type: link_type)
+
+      if linked_items.empty?
+        error_details = {
+          error: {
+            code: 404,
+            message: "Could not find any item with a link of type '#{link_type}' linked to this item: '#{content_id}'"
+          }
+        }
+
+        raise CommandError.new(code: 404, error_details: error_details)
```

Comment:
This doesn't look quite correct. It's not differentiating between the case where no content item exists with the given content_id, and the content item exists, but has no links of the given type. The latter should probably return empty array, not a 404.